src/brevitas/core/quant/float.py

```diff
@@ -13,6 +13,9 @@
 from brevitas.function.ops import max_float
 from brevitas.function.ops_ste import floor_ste
 
+# max int that can be passed to torch.exp2() without running into inf
+MAX_REPRESENTABLE_INT = 127
```

Comment:
I am changing my mind about this. Maybe we can handle the `inf` case in the tests somehow? As expected results?